Taarafo.Core.Tests.Unit/Services/Foundations/Comments/CommentServiceTests.Exceptions.Modify.cs

```diff
@@ -213,5 +215,52 @@ public async Task ShouldThrowServiceExceptionOnModifyIfServiceErrorOccursAndLogI
             this.storageBrokerMock.VerifyNoOtherCalls();
             this.loggingBrokerMock.VerifyNoOtherCalls();
         }
+
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnModifyIfReferenceErrorOccursAndLogItAsync()
+        {
+            // given
+            int minuteInPast = GetRandomNegativeNumber();
```

Comment:
The tests scope if just for handling exception so we need not prepare the test data to pass the valdations. Please check the above tests. Also, this test should be second in the order of this file.

Reply:
@ShriHumrudha have updated all the tests.